docs/application/native/guides/app-management/overview.md

```diff
@@ -74,6 +74,10 @@ You can use the following application data exchange features in your native appl
 
   You can display Tizen widget applications, manage life-cycle of its instances, and retrieve widget information.
 
+- [Cion](cion.md)
```

Comment:
I think Cion should be placed between [Application Controls] and [Common Application Controls] to maintain alphabetical order.
@mijin85cho can you please confirm this?

Reply:
it's moved